frontend/src/components/EnvironmentConfiguration.jsx

```diff
@@ -756,11 +849,80 @@ export default function EnvironmentConfiguration (env) {
+                                {sadeCount > 1 && <Button style={{fontSize:'15px'}} onClick={handleDecrement}>-</Button>}
+                                {sadeCount && <Button style={{fontSize:'15px'}} variant="contained" color="warning">{sadeCount}</Button>}
+                                <Button style={{fontSize:'15px'}} onClick={handleIncrement} disabled={sadeCount===10}>+</Button>
+                            </ButtonGroup>
+                        </Grid>
+                    </Grid>
+
+                    {sadeArray.map((sade, index) => (
+                        <Accordion key={index} classes={{root:classes.transparentBackground}}>
+                            <AccordionSummary
+                                expandIcon={<ExpandMore />}
+                                aria-controls="panel1a-content"
+                                id="panel1a-header"
+                                sx={{backgroundColor: '#643E05'}}
+                            >
+                                <Box sx={{display:'flex', alignItems:'center', justifyContent:'space-between', width:'100%'}}>
+                                    <Typography variant="h5" sx={{color:'#F5F5F5', pb:1}}>
+                                        {sade.sadeName}
+                                    </Typography>
+                                </Box>
+                            </AccordionSummary>
+                            <AccordionDetails sx={{backgroundColor:'#75531E47'}}>
+                                <Grid container spacing={2}>
+                                    {[
+                                        {label:'Name', key:'sadeName', type:'text'},
+                                        {label:'Height', key:'height', type:'number'},
+                                        {label:'Latitude 1', key:'latitude1', type:'number'},
+                                        {label:'Longitude 1', key:'longitude1', type:'number'},
+                                        {label:'Latitude 2', key:'latitude2', type:'number'},
+                                        {label:'Longitude 2', key:'longitude2', type:'number'},
+                                        {label:'Latitude 3', key:'latitude3', type:'number'},
+                                        {label:'Longitude 3', key:'longitude3', type:'number'},
+                                        {label:'Latitude 4', key:'latitude4', type:'number'},
+                                        {label:'Longitude 4', key:'longitude4', type:'number'},
+                                    ].map((field, i) => (
+                                        <Grid item xs={6} key={i}>
+                                            <StyledInputLabel id={field.key}>{field.label}</StyledInputLabel>
+                                            <TextField
+                                                sx={{
+                                                    backgroundColor:'#71665E',
```

Comment:
@Hoshi-Okami The values I entered into the text box are not appearing. The text box remains empty.